### PR TITLE
PP-5534 Update internal states

### PR DESCRIPTION
**Pull request by danworth**
Replace `EXPIRED` with `USER_SETUP_EXPIRED`
Remove conditions and tests for `PENDING` and `USER_CANCEL_NOT_ELIGBLE` as
the internal states no longer exists.
Renamed `SUBMITTED` to `SUBMITTED_TO_PROVIDER` and added new internal
state of `SUBMITTED_TO_BANK` with tests.
Align heading and message for `ACTIVE` with submitted statuses.

### CODE REVIEW

**Comment by stephencdaly** [comment]
We seem to have lost the error message for `USER_SETUP_CANCELLED` I think we should show what we have for `USER_CANCEL_NOT_ELIGIBLE` before?

**Comment by danworth** [comment]
> We seem to have lost the error message for `USER_SETUP_CANCELLED` I think we should show what we have for `USER_CANCEL_NOT_ELIGIBLE` before?

Sure will do, good suggestion!

File: common/middleware/mandate-state-enforcer/mandate-state-enforcer.js [hunk 1]
```diff
@@ -11,37 +11,31 @@ const pageToValidMandateStateMap = {
...
-    includeReturnUrl: false
-  },
-  PENDING: {
-    heading: 'Your Direct Debit mandate is being processed',
-    message: 'Check your confirmation email for details of your mandate.',
-    includeReturnUrl: false
-  },
+  SUBMITTED_TO_PROVIDER: inProgressMessage,
+  SUBMITTED_TO_BANK: inProgressMessage,
+  ACTIVE: inProgressMessage,
```
Comment (stephencdaly) [inline common/middleware/mandate-state-enforcer/mandate-state-enforcer.js]: Should other mandate states past active also show this? It's probably not so important as a user shouldn't be looking at this page again. I.e. CANCELLED, EXPIRED

Comment (danworth) [reply common/middleware/mandate-state-enforcer/mandate-state-enforcer.js]: hmm I know we considered it, but as you say thought it very unlikely that a user would come back to this page that far into the lifecycle of a mandate; the downside would be having to list all of our other internal states held in DDConnector in this map (or change the implementation) which makes things more tightly coupled than I think is necessary. So on balance I don't think we should. Also, I'm not sure that message works for the other states, if its cancelled I think saying 'Your Direct Debit mandate is being processed` is a bit misleading.

Comment (stephencdaly) [reply common/middleware/mandate-state-enforcer/mandate-state-enforcer.js]: i think that's fair enough, we wouldn't expect these statuses to occur until way down the line anyway

**Pull request merged**